src/core/annotation.js

```diff
@@ -471,6 +476,7 @@ class Annotation {
    */
   setAppearance(dict) {
     this.appearance = null;
+    this.checkAppearance = null;
```

Comment:
I think this should be `this.checkedAppearance = null;` given the other occurrences.